docs/certificate_management.md

```diff
@@ -128,7 +128,7 @@ When using cert-manager as the certificate manager for Open Service Mesh, it wil
 To get the certificate issued from cert-manager, OSM controller will create a certificate request from `crypto/x509`:
 
 ```go
-x509.CertificateRequest{
+x509.*CertificateReques{
```

Comment:
If I'm being pedantic (and fixing a typo):
```suggestion
&x509.CertificateRequest{
```

(Also applies to the next two changed lines in this file)

Reply:
good catch, fixed!